pkg/virt-handler/virtwrap/manager.go

```diff
@@ -328,21 +327,14 @@ func VMNamespaceKeyFunc(vm *v1.VM) string {
 	return domName
 }
 
-func (l *LibvirtDomainManager) SyncVM(vm *v1.VM) (*api.DomainSpec, error) {
-	var wantedSpec api.DomainSpec
-	mappingErrs := model.Copy(&wantedSpec, vm.Spec.Domain)
-
-	if len(mappingErrs) > 0 {
-		return nil, errors.NewAggregate(mappingErrs)
-	}
-
+func (l *LibvirtDomainManager) SyncVM(vm *v1.VM, domDesign *designer.DomainDesign) (*api.DomainSpec, error) {
```

Comment:
Sorry, but you are predicting, that there will be cases where we will see issues with network devices, that is not the same as concrete examples. I am not saying that they will not happen. But it should be possible to point concrete examples out (e.g. point to fields in the xmls or something like that). Just give me concrete examples where I can see that it is not possible for thinks we want to provide with KubeVirt. I don't think that I am asking for too much. I want/need to learn about such cases, to better understand our application domain.